python/src/spark_rapids_ml/params.py

```diff
@@ -101,14 +101,14 @@ def _param_mapping(cls) -> Dict[str, Optional[str]]:
     @classmethod
     def _param_value_mapping(
         cls,
-    ) -> Dict[str, Callable[[str], Union[None, str, float, int]]]:
+    ) -> Dict[str, Callable[[Any], Union[None, str, float, int]]]:
```

Comment:
Wonder if we should update all other classes to reflect this signature change, just for consistency?  (It might clutter this PR though).

Reply:
One line change seems fine. Updated in this PR.